code/__DEFINES/conflict.dm

```diff
@@ -61,21 +61,29 @@
 //flags_gun_features
 #define GUN_CAN_POINTBLANK (1<<0)
 #define GUN_UNUSUAL_DESIGN (1<<1)
-#define GUN_SHOTGUN_CHAMBER (1<<2)
-#define GUN_INTERNAL_MAG (1<<3)
-#define GUN_AUTO_EJECTOR (1<<4)
-#define GUN_AMMO_COUNTER (1<<5)
-#define GUN_LOAD_INTO_CHAMBER (1<<6)
-#define GUN_ENERGY (1<<7)
-#define GUN_WIELDED_FIRING_ONLY (1<<8)
-#define GUN_ALLOW_SYNTHETIC (1<<9)
-#define GUN_WIELDED_STABLE_FIRING_ONLY (1<<10)
-#define GUN_IFF (1<<11)
-#define GUN_DEPLOYED_FIRE_ONLY (1<<12)
-#define GUN_IS_ATTACHMENT (1<<13)
-#define GUN_ATTACHMENT_FIRE_ONLY (1<<14)
-#define GUN_IS_SENTRY (1<<15)
-#define GUN_PUMP_REQUIRED (1<<16)
+#define GUN_AMMO_COUNTER (1<<2)
+#define GUN_WIELDED_FIRING_ONLY (1<<3)
+#define GUN_ALLOW_SYNTHETIC (1<<4)
+#define GUN_WIELDED_STABLE_FIRING_ONLY (1<<5)
+#define GUN_IFF (1<<6)
+#define GUN_DEPLOYED_FIRE_ONLY (1<<7)
+#define GUN_IS_ATTACHMENT (1<<8)
+#define GUN_ATTACHMENT_FIRE_ONLY (1<<9)
+#define GUN_IS_SENTRY (1<<10)
+#define GUN_ENERGY (1<<11)
+#define GUN_AMMO_COUNT_BY_PERCENTAGE (1<<12)
+
+//reciever_flags. Used to determin how the gun cycles, what kind of ammo it uses, etc.
+#define RECIEVER_REQUIRES_OPERATION (1<<0)
+#define RECIEVER_MAGAZINES (1<<1)
+#define RECIEVER_HANDFULS (1<<2)
+#define RECIEVER_TOGGLES (1<<3)
+#define RECIEVER_CLOSED (1<<4)
+#define RECIEVER_CYCLES (1<<5)
+#define RECIEVER_AUTO_EJECT (1<<6)
+#define RECIEVER_TOGGLES_EJECTS (1<<7)
+#define RECIEVER_LOCKS (1<<8)
+#define RECIEVER_NO_HANDFULS (1<<9)
```

Comment:
rename these please, no one needs to know what is a receiver to understand what this means